devsupApp/src/Makefile

```diff
@@ -9,6 +9,9 @@ include $(TOP)/configure/CONFIG_PY
 INSTALL_SHRLIB = $(PY_INSTALL_DIR)/devsup
 
 LOADABLE_LIBRARY_HOST += _dbapi
+ifeq ($(OS),Windows_NT)
+    SHRLIB_SUFFIX_BASE = .pyd
```

Comment:
I think this should be in `configure/CONFIG_PY`, where the equivalent for Darwin/OSX is set.

```make
ifeq ($(OS_CLASS),WIN32)
LOADABLE_SHRLIB_SUFFIX = .pyd
endif
```

Reply:
OK, but I get a build error - which I can resolve by setting:
LOADABLE_LIBRARY_HOST_DEFAULT += _dbapi
LOADABLE_LIBRARY_HOST_WIN32 += _dbapi.pyd

Reply:
It does need to be SHRLIB_SUFFIX_BASE (setting LOADABLE_SHRLIB_SUFFIX does not have the desired effect).
But this can be, as you suggest, in CONFIG.PY.